Library/Formula/teapot.rb

```diff
@@ -1,17 +1,23 @@
-require 'formula'
-
 class Teapot < Formula
-  homepage 'http://www.syntax-k.de/projekte/teapot/'
-  url 'http://www.syntax-k.de/projekte/teapot/teapot-2.3.0.tar.gz'
-  sha1 'cac70c7967ba72166cdbd1806b674cd8299399e7'
+  homepage "http://www.syntax-k.de/projekte/teapot/"
+  url "http://www.syntax-k.de/projekte/teapot/teapot-2.3.0.tar.gz"
+  sha256 "580e0cb416ae3fb3df87bc6e92e43bf72929d47b65ea2b50bc09acea3bff0b65"
 
-  depends_on 'cmake' => :build
+  depends_on "cmake" => :build
 
   def install
-    args = std_cmake_args + ['-DENABLE_HELP=OFF', '..']
-    mkdir 'macbuild' do
-      system 'cmake', *args
-      system 'make install'
+    args = std_cmake_args + ["-DENABLE_HELP=OFF", ".."]
+    mkdir "macbuild" do
+      system "cmake", *args
+      system "make", "install"
     end
   end
+
+  def caveats; <<-EOS.undent
+    Teapot documentation does not get built because it requires software (LyX)
+    not included in Homebrew. For a good introduction to Teapot, see:
+
+        https://kmandla.wordpress.com/2010/08/11/how-to-use-teapot-like-a-pro/
```

Comment:
Let's drop this (I will kill this when merging).

IMHO, caveats should only include Homebrew specific information. Users should find general information such as document on their own. And it's pretty bad to include unofficial third party document.